DEPS

```diff
@@ -0,0 +1,76 @@
+vars = {
+  'chromium_version':
+    '63.0.3239.150',
+  'libchromiumcontent_revision':
+    '0e8b7216fe616405ea14ff5bece3ca087ead613c',
+  'node_version':
+    'v9.7.0-33-g538a5023af',
+  'native_mate_revision':
+    '4cd7d113915de0cc08e9a218be35bff9c7361906',
+
+  'chromium_git':
+    'https://chromium.googlesource.com',
+
+  'electron_git':
+    # TODO(nornagon): once
+    # https://github.com/electron/libchromiumcontent/pull/536 lands, switch
+    # this to https://github.com/electron.
+    'https://github.com/nornagon',
```

Comment:
Can we change it to `https://github.com/electron` now that the libcc PR is merged.